web/app.php

```diff
@@ -6,7 +6,7 @@
 $loader = require __DIR__.'/../app/autoload.php';
 include_once __DIR__.'/../var/bootstrap.php.cache';
 
-$kernel = new AppKernel('prod', false);
+$kernel = new AppKernel('dev', false);
```

Comment:
why do you change the `app.php` env? The Symfony 3.4 best practice defines a prod env in `app.php` and a dev env in `app_dev.php`.
Just use the file you need to launch the env you want